src/api/Shrooms.Contracts.DataTransferObjects/Models/Wall/WallDto.cs

```diff
@@ -16,6 +16,8 @@ public class WallDto
 
         public string Description { get; set; }
 
+        public bool IsHidden { get; set; }
```

Comment:
`IsHidden` implies that it is hidden from everywhere, the better name could be `IsHiddenFromAllWalls` or `IsHiddenFromNonSubscribers`